### DISCUSSION TITLE
How to try out the latest builds?

### DISCUSSION THREAD

**Discussion by saadulkh** [nyx-lyb3ra/Replay#165] [Q&A] [answered]
It used to be there in the actions manifests before but not now. Please let me know where I can get the up-to-date flatpaks.

**Comment by saadulkh** [comment]
Here (for now): https://github.com/nahuelwexd/Replay/actions?query=workflow:Flatpak

**Comment by nyx-lyb3ra** [comment, marked as answer]
@saadulkh I need to update the README, but now there is a small Flatpak repo

```sh
flatpak remote-add --if-not-exists replay-nightly https://nahuelwexd.github.io/replay-nightly/replay-nightly.flatpakrepo

flatpak install replay-nightly app.drey.Replay.Devel//master
```
